test/functional/ui/completion_spec.lua

```diff
@@ -0,0 +1,41 @@
+local helpers = require('test.functional.helpers')
```

Comment:
please move this file to `test/functional/viml/completion_spec.lua`. And remove the Screen import. Screen tests are UI tests, and they are generally separate from non-UI tests.